src/ui/public/agg_types/buckets/ip_range.js

```diff
@@ -15,10 +15,9 @@ export default function RangeAggDefinition(Private) {
     createFilter: createFilter,
     getKey: function (bucket, key, agg) {
       if (key) return key;
-
-      const from = _.get(bucket, 'from', '*');
-      const to = _.get(bucket, 'to', '*');
-      return `${from}-${to}`;
+      const from = _.get(bucket, 'from', '-Infinity');
+      const to = _.get(bucket, 'to', 'Infinity');
+      return `${from} to ${to}`;
```

Comment:
I just discovered open ended ip range filters fail in 4.x as well, so this is probably a good change to make regardless and backport it. Still curious if there are any BC concerns other than just the label changing in the UI?